src/main/java/mate/academy/dto/UpdateBookRequestDto.java

```diff
@@ -2,13 +2,25 @@
 
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.validator.Author;
+import mate.academy.validator.CoverImage;
+import mate.academy.validator.Description;
+import mate.academy.validator.Isbn;
+import mate.academy.validator.Price;
+import mate.academy.validator.Title;
 
 @Data
 public class UpdateBookRequestDto {
+    @Title
```

Comment:
same here
You can even use for example
`@Size(min = 10, max = 13, message = "Required length 10-13 digits")`